docker-compose.yml

```diff
@@ -4,7 +4,7 @@ version: '3'
 # Only one of these services should be run at a time.
 services:
   gitlab-ce:
-    image: gitlab/gitlab-ce
+    image: gitlab/gitlab-ce:14.9.0-ce.0
```

Comment:
Will dependabot pick up the patch releases?

Reply:
@jtymes Do you know a way to find it out?

Reply:
Sure, just was curious. :shipit: